src/cc/bcc_elf.c

```diff
@@ -0,0 +1,269 @@
+#include <sys/types.h>
```

Comment:
I don't mind the kernel style for .c files either, but I would really rather use it only when in a pure C project. Mixing C and C++ files with different formats is more aggravating that suffering with a lesser format, IMO.